src/Layer/C3DTilesLayer.js

```diff
@@ -253,6 +102,68 @@ class C3DTilesLayer extends GeometryLayer {
             }
         }
     }
+
+    // Finds the batch table of an object in a 3D Tiles layer. This is
+    // for instance needed when picking because we pick the geometric
+    // object which is not at the same level in the layer structure as
+    // the  batch table. More details here on itowns internal
+    // organization of 3DTiles:
+    // https://github.com/MEPP-team/RICT/blob/master/Doc/iTowns/Doc.md#itowns-internal-organisation-of-3d-tiles-data
+    findBatchTable(object) {
+        if (object.batchTable) {
+            return object.batchTable;
+        }
+        if (object.parent) {
+            return this.findBatchTable(object.parent);
+        }
+        return undefined;
+    }
+
+    getInfoFromIntersectObject(intersects) {
+        const resultInfo = {};
+
+        let batchID = -1;
+        let batchTable = {};
+        // First, we get the ID and the batch table of the intersected object.
+        // (the semantic information about a feature is located in its batch
+        // table (see 3D Tiles specification).
+        for (let i = 0; i < intersects.length; i++) {
+            // interAttributes are glTF attributes of b3dm tiles (i.e.
+            // position, normal, batch id)
+            const interAttributes = intersects[i].object.geometry.attributes;
+            if (interAttributes) {
+                const batchIDs = interAttributes._BATCHID || interAttributes.batchId;
```

Comment:
there is 2 attributes names (`_BATCHID` and `batchId`)  for batch id in the specification ?

Reply:
To my knowledge there is only `_BATCHID` but you added `batchId` in [this commit](https://github.com/iTowns/itowns/commit/7b067c7deb21614b731c90e8486ac2fffbcb5e82#diff-276f2190b0230ea4d24aeca3427e133eR25), so I thought there was a good reason to that ? If no, I will remove it since I tried it without `batchId` and it works.

Reply:
ok, I'll look into it. We'll leave it like this for now.